dbc2val/mapping.yml

```diff
@@ -98,51 +181,94 @@ CP_hvChargeStatus:
           CP_EVSE_TEST_PASSED: "true"
           CP_EXT_EVSE_TEST_ACTIVE: "true"
 
-
-
-VCFRONT_brakeFluidLevel:
+# IGBT temperature is not "Motor" temperature, but let's use it, as that might be the more critical temp anyway
+PTC_rightTempIGBT:
+  vss: # https://github.com/COVESA/vehicle_signal_specification/blob/master/spec/Powertrain/ElectricMotor.vspec
+    datatype: int16
+    type: sensor
+    unit: celsius
+    description: Motor temperature.
+  databroker:
+    datatype: 3 # FLOAT number taken from types_pb2.py
+    changetype: 2 # ON_CHANGE number taken from types_pb2.py
   minupdatedelay: 1000
   targets:
-    Vehicle.Chassis.Axle.Row1.Wheel.Left.Brake.FluidLevelLow:
-      transform:
-        fullmapping:
-          LOW: "true"
-          NORMAL: "false"
-    Vehicle.Chassis.Axle.Row1.Wheel.Right.Brake.FluidLevelLow:
-      transform:
-        fullmapping:
-          LOW: "true"
-          NORMAL: "false"
-    Vehicle.Chassis.Axle.Row2.Wheel.Left.Brake.FluidLevelLow:
-      transform:
-        fullmapping:
-          LOW: "true"
-          NORMAL: "false"
-    Vehicle.Chassis.Axle.Row2.Wheel.Right.Brake.FluidLevelLow:
-      transform:
-        fullmapping:
-          LOW: "true"
-          NORMAL: "false"
+    Vehicle.Powertrain.ElectricMotor.Motor.Temperature:
+      transforn: {}
+
+# Mapping Parking brake
```

Comment:
I think this file is broken/regressed. A lot of stuff is commented out

(and we would still have the option to add more mapping files)

Reply:
I commented everything out which is not in VSS Spec available
https://github.com/COVESA/vehicle_signal_specification/blob/master/spec/Vehicle/Vehicle.vspec

Also the Vehicle Speed was given twice. One time in Vehicle.Speed and the ohter one in Vehicle.OBD.Speed with the same resolution and unit.
As far as i remember the VSS was founded to have only one signal name for a signal over the whole car. 
So we should decide which Vehicle Speed should stay in the Spec.

Reply:
Hi, I think then you used another too old/to new VSS Version.

The existing mapping files only contains elements available in the released VSS version (2.2, https://github.com/COVESA/vehicle_signal_specification/releases/tag/v2.2), but does not yet  include anything form upcoming 3.0 (not yet released, some stuff will move then. Would update the mapping once 3.0 is released).

Specific example Vehicle.Speed and Vehicle.OBD.Speed do both exist, as for historical reasons the OBD branch has been there since the "dark times " (e.g. before my time even :) ), and models everything from OBD.  
(In general it is in the mapping to show you can map one signal to different targets. More useful examples would be gear and parking brake in this example or (not in the example) FuelSystem.Range vs PowerTrain.Range on a on hybrid)

In any case we should have the (most complete) mapping to current released VSS version. If for some reason another example uses an older or newer/unreleased version maybe we should provide several mappings, i.e. mapping_dogmodeexamaple.yml etc. (and make to have an equivalent line in the mappings like now, where it says `This maps to VSS 2.2`)